scripts/helpers/changeloglink.js

```diff
@@ -1,13 +1,11 @@
 'use strict'
 
-const semver = require('semver');
-
+const semver = require('semver')
 
 module.exports = function (version) {
+  if (!version) { return '' }
 
-    if (!version) { return ''; }
-
-    return semver.gte(version, '1.0.0') ?
-        `https://github.com/nodejs/io.js/blob/${version}/CHANGELOG.md` :
-        `https://github.com/joyent/node/blob/${version}/ChangeLog`;
-};
+  return semver.gte(version, '1.0.0')
+    ? `https://github.com/nodejs/io.js/blob/${version}/CHANGELOG.md`
+    : `https://github.com/joyent/node/blob/${version}/ChangeLog`
```

Comment:
Yes, the script has been used the last time before the v4.0.0 release. Fixing this …